luatest/hooks.lua

```diff
@@ -43,6 +67,11 @@ local function define_hooks(object, hooks_type)
             end
         end
     end
+    if preloaded_hook then
+        for _, v in ipairs(preloaded_hook) do
+            table.insert(hooks, v)
+        end
+    end
```

Comment:
Isn't it the same as if we do `local hooks = table.copy(preloaded_hook)` in the first line of the function?

Reply:
Removed.